dcp_prototype/backend/browser/code/common/s3_utils.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+def generate_file_url(file_prefix, expiration=3600):
```

Comment:
@Bento007 When a user clicks "Download matrix", a download URL with 24hr expiration will be generated and returned. Is this sufficient for our registered access requirements?

Reply:
Yep, that's fine for the time being